src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -132,12 +132,13 @@ void regmodule_offline_transformations(py::module m) {
 
     m_offline_transformations.def(
         "paged_attention_transformation",
-        [](std::shared_ptr<ov::Model> model) {
+        [](std::shared_ptr<ov::Model> model, bool use_cache_eviction) {
```

Comment:
This mode itself doesn't mean that PA uses cache eviction: it just adds scores for tokens and allows unaligned block tables. These are two different modes for the transformation that technically can be used independently. I would suggest to use two options instead of one. One controls unaligned block indices for layers, another one controls extra outputs for PA. This transformation is low-level enough to have these two options separate.

Reply:
done